src/main/java/frc/robot/commandGroups/CommandGroups.java

```diff
@@ -127,17 +114,11 @@ public Command setHood() {
     }
 
     public Command outtakeGripper() {
-        return elevator.setHeight(CommandGroupsConstants.OUTTAKE_HEIGHT)
-                .onlyIf(gripper::isGripperNearFoldedPosition)
-                .andThen(gripper.outtake())
-                .withName("outtakeGripper");
+        return gripper.outtake().withName("outtakeGripper");
```

Comment:
IMO it's better to delete this function, it just calls another function. You can use `gripper.outtake()` where needed. `withName()` is also redundant here, but I would improve the name set inside `Gripper`'s `outtake` method, it's a bit too general currently.

Reply:
cc162cb3745de97f986f9ce499557360459d4546

Reply:
Nice, and about the second thing?